Silicon/CommonSocPkg/Library/WatchDogTimerLib/WatchDogTimerLib.c

```diff
@@ -116,6 +116,25 @@ WdtClearFlags (
   IoWrite32 (WdtGetAddress (), Readback);
 }
 
+/**
+  Set WDT TCC flags in scratchpad
+
+  @param[in] Flags             The scratchpad flags that would be set.
+
+**/
+VOID
+EFIAPI
+WdtSetTccScratchpad (
```

Comment:
WdtDogTimerLib is for common usage, and it should not be specific to TCC.
Please keep the API interface generic and use the flag to indicate TCC use cases.

Reply:
rename to WdtSetScratchpad() to keep common usage and focus on scratchpad only